keepvariable/keepvariable_core.py

```diff
@@ -587,17 +612,42 @@ def query(
                 value_str = "|".join(values)
                 final_query += tag_query_template.format(field=field, value=value_str)
 
+        # If no query was specified, return all records from the index
+        if not final_query:
```

Comment:
i would compare it to empty string

Reply:
Yes - makes it clear that it is always string

Reply:
this looks like bool